app/src/androidTest/java/com/github/se/cyrcle/ui/theme/molecules/TopAppBarTest.kt

```diff
@@ -76,4 +76,28 @@ class TopAppBarTest {
         .assertIsDisplayed()
         .assertHasClickAction()
   }
+
+  @Test
+  fun displayTopAppBarWithLongTitle() {
+    val tag2 = "TopAppBar2"
+    val title =
+        "This is a very long title that should be truncated but just in case it is not, I am making it longer"
+    composeTestRule.setContent {
+      Scaffold(topBar = { TopAppBar(navigationActions, title, tag2) }) { innerPadding ->
+        Column(modifier = Modifier.padding(innerPadding)) {}
+      }
+    }
+
+    // Top App Bar
+    composeTestRule.onNodeWithTag(tag2).assertIsDisplayed()
+
+    // Title
+    composeTestRule.onNodeWithTag("${tag2}Title", true).assertIsDisplayed().assertTextEquals(title)
```

Comment:
Just a little clarification here, the test is supposed to verify that the title was truncated but here it tests that the topAppBar title is the original non truncated. Am I missing something ?

Reply:
The whole point of the feature is not to truncate the title. I think what mislead you is the content of the title but consider it as "would be truncated without this feature" rather than "should be truncated"

Reply:
Ohhhh makes sense thanks ! It might have something to do with the title yes.